lib/open_food_network/bulk_coop_report.rb

```diff
@@ -153,7 +153,7 @@ def customer_payments_amount_paid(line_items)
       line_items.map(&:order).uniq.sum(&:payment_total)
     end
 
-    def empty_cell(line_items)
+    def empty_cell(*)
```

Comment:
I didnt know this existed in Ruby :see_no_evil: 
I think I would prefer to see an indication of what could be sent and not used here. So _line_items would be better than * imho.